astroquery/jplhorizons/core.py

```diff
@@ -792,7 +792,7 @@ def elements_async(self, get_query_payload=False,
 
         return response
 
-    def vectors_async(self, get_query_payload=False,
+    def vectors_async(self, get_query_payload=False, refplane='ecliptic',
```

Comment:
please add the new kwarg to the end of the signature so to avoid breaking the current API